README.md

```diff
@@ -105,59 +100,57 @@ In each class (lecture and tutorial) there will be an assignment:
+| 4    | Effective data visualization                     | Expand your data visualization knowledge and tool set beyond what we have seen and practiced so far. We will move beyond scatter plots and learn other effective ways to visualize data, as well as some general rules of thumb to follow when creating visualations. All visualization tasks this week will be applied to real world data sets. Again, this will be in the context of a real world data science application and we will continue to practice working through a whole case study that includes downloading data from the web, wrangling the data into a useable format and creating an effective data visualization. |
+| 5    | Version control                                  | Collaboration with version control                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   |
+| 5    | Group contract due                               |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      |
+| 6    | Classification                                   | Introduction to classification using K-nearest neighbours (k-nn)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     |
```

Comment:
@joelostblom the explanations of the material starting in week 5 gets very bare compared to weeks 1-4. Maybe add some more text here?